tests/settings.py

```diff
@@ -24,7 +26,7 @@
 
     'rest_framework',
 
-    'katka',
+    'katka.apps.KatkaCoreConfig',
```

Comment:
You can circumvent having to do this: https://docs.djangoproject.com/en/2.2/ref/applications/#for-application-authors , because otherwise we would also have to do this when using this package right?

Reply:
yeah, I discussed that before with @rodgomes, but see in a couple of sentences below: 

> Besides this use case, it’s best to avoid using default_app_config and instead specify the app config class in INSTALLED_APPS as described next.

And a little bit above the section that you link:
> New applications should avoid default_app_config. Instead they should require the dotted path to the appropriate AppConfig subclass to be configured explicitly in INSTALLED_APPS.

so we decided to follow that and tell people to configure it explicitly.

Reply:
Okay :) I see, I didn't notice. Still seems odd that they suggest this, forcing package users to know which Config they need.